### PR TITLE
make consumable via npm

### PR DESCRIPTION
**Pull request by mdarens**
- use webpack css-loader to load css
- add linting

The repo should be renamed to knockout.pickadate after this is merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,21 @@
 # Knockout Pickadate Binding
 
-> Simple knockout binding that wraps [pickadate.js](http://amsul.ca/pickadate.js).
+> Simple knockout binding which wraps [pickadate.js](http://amsul.ca/pickadate.js).
```
Comment (chadly) [inline README.md:3]: grammar

File: package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "url": "https://github.com/civicsource/knockout.pickadate/issues"
+  },
+  "homepage": "https://github.com/civicsource/knockout.pickadate#readme",
+  "devDependencies": {
+    "babel-eslint": "6.x",
+    "eslint": "2.x",
+    "eslint-plugin-filenames": "1.x"
+  },
+  "dependencies": {
+    "css-loader": "^0.23.1",
```
Comment (chadly) [inline package.json]: nah to this

**Comment by chadly** [comment]
repo is renamed

File: README.md (hunk 1 above)
Comment (chadly) [inline README.md:3]: > binding that which will once wrap

**Comment by chadly** [comment]
:shipit:

File: README.md (hunk 1 above)
Comment (mdarens) [inline README.md:3]: one does not simply use restrictive clauses all willy-nilly

**Pull request merged**